system/jlib/jsocket.cpp

```diff
@@ -2662,7 +2687,7 @@ CSocket::CSocket(const SocketEndpoint &ep,SOCKETMODE smode,const char *name)
         SocketEndpoint self;
         self.setLocalHost(0);
         self.getUrlStr(hostname);
-        setTraceName("S>", hostname);
+        setTraceName("S>", hostname.str());
```

Comment:
fine, but not strictly necessary, const char * cast operator will call str() automatically.